cpp/src/Slice/SliceUtil.cpp

```diff
@@ -483,7 +483,15 @@ Slice::checkIdentifier(const string& id)
         name = id;
     }
 
+    // Check that the identifier isn't empty.
+    if (name.empty())
+    {
+        currentUnit->error("missing identifier: illegal empty identifier");
```

Comment:
```suggestion
        currentUnit->error("missing identifier: invalid empty identifier");
```

Reply:
This message was removed entirely.
It should now be impossible for this function to ever have an empty identifier.